foundry.toml

```diff
@@ -5,6 +5,9 @@ libs = ["lib"]
 
 # Compiler settings
 solc = "0.8.28"
+evm_version = "cancun"
+bytecode_hash = "none"
+cbor_metadata = false
```

Comment:
Added some compiler configs to make sure the CREATE2 is deterministic. Followed the recommendations in https://book.getfoundry.sh/guides/deterministic-deployments-using-create2